File: C Sharp/pot.cs
using System;

public class pot {
    public static void Main (string[] args) {
        int cases = int.Parse (Console.ReadLine ());
        int sum = 0;
        while (cases-- > 0) {
            string line = Console.ReadLine ();
            int num = int.Parse (line.Substring (0, line.Length - 1));
            int pow = int.Parse (line.Substring (line.Length - 1));
            sum += (int) Math.Pow (num, pow);
        }
        Console.WriteLine (sum);
    }
}

File: Java/pot.java

//  https://open.kattis.com/problems/pot
import java.util.Scanner;

public class pot {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cases = sc.nextInt();
        int total = 0;
        sc.nextLine();

        while (cases-- > 0) {
            String line = sc.nextLine();
            int num = Integer.parseInt(line.substring(0, line.length() - 1));
            int power = Integer.parseInt(line.substring(line.length() - 1));
            total += Math.pow(num, power);
        }
        System.out.println(total);
        sc.close();
    }
}